Feature/user inputs3

Samylms opened the pull request:
Latest upddate keylogger

Commit:
Update keylogger

Commit:
Update keylogger

Commit:
keylogger update

Commit:
keylogger update

Commit:
Merge remote-tracking branch 'remotes/origin/master' into feature/user-inputs3

Commit:
keylog update

Commit:
keylog update

Commit:
Merge remote-tracking branch 'remotes/origin/master' into feature/user-inputs3

# Conflicts:
#	app/src/main/java/ca/uqac/bigdataetmoi/ui/keylogger/KeyloggerFragment.java
#	app/src/main/java/ca/uqac/bigdataetmoi/workers/Keylogger.java
#	app/src/main/res/values/colors.xml

papychampi committed:
Merge branch 'master' into feature/user-inputs3

papychampi reviewed (changes requested):
Problèmes lors de l'utilisation de la feature...

Lorsque l'on arrive sur la page du keylogger on est embarqué sans savoir pourquoi dans les paramètres du téléphone

Même quand on a activé le service on a l'écran qui demande d'activer le service et qui nous redirige vers l'écran des paramètres du téléphone

itorterat committed:
Fix : Fix pb de crash au premier lancement sur apps (#26)

Commit:
update keylogger

Commit:
update keylogger

antoadam committed:
feat(user): delete user and stop workers (#29)

papychampi committed:
Feature/wifi Corrections (#28)

* Adding Wifi List

* Upload de wifis Op

* wip

* Remove comment Position Worker

* Correction Position Worker

* Correct worker

* Correction Merge

* stash

JeromeIsoard committed:
Fix/message (#27)

* feature message

* feature message

* - retrieve message from DB
- display messages on the fragment
- retrieve messages from sms Content
- save messages in DB

* - explanation message fragment
- reverse message list

* - suppression fichiers

* Worker starting date correction

* Worker starting date correction2

Commit:
Auto stash before rebase of "origin/master"

Commit:
Merge remote-tracking branch 'origin/feature/user-inputs3' into feature/user-inputs3

Samylms reviewed (commented):
feature keylogger
